### PR TITLE
[BEAM-845] Update Apex runner info after merge to master.

### PR DESCRIPTION
**Pull request by tweise**
@davorbonaci please review

### CODE REVIEW

**Comment by jbonofre** [comment]
CC: @jbonofre

File: src/documentation/runners/apex.md [hunk 1]
```diff
@@ -5,5 +5,5 @@ permalink: /documentation/runners/apex/
 ---
 # Using the Apache Apex Runner
 
-This page is under construction ([BEAM-825](https://issues.apache.org/jira/browse/BEAM-825)). The runner is on a feature branch.
+This page is under construction ([BEAM-825](https://issues.apache.org/jira/browse/BEAM-825)).
```
Comment (davorbonaci) [inline src/documentation/runners/apex.md]: FYI only -- see #76 and #77 for good examples on getting started on this. (Not for this PR.)

File: src/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ The Apache Beam project is in the process of bootstrapping. This includes the we
 
 # Apache Beam (incubating)
 
-Apache Beam is an open source, unified programming model that you can use to create a data processing **pipeline**. You start by building a program that defines the pipeline using one of the open source Beam SDKs. The pipeline is then executed by one of Beam's supported **distributed processing back-ends**, which include [Apache Flink](http://flink.apache.org), [Apache Spark](http://spark.apache.org), and [Google Cloud Dataflow](https://cloud.google.com/dataflow).
+Apache Beam is an open source, unified programming model that you can use to create a data processing **pipeline**. You start by building a program that defines the pipeline using one of the open source Beam SDKs. The pipeline is then executed by one of Beam's supported **distributed processing back-ends**, which include [Apache Apex](http://apex.apache.org), [Apache Flink](http://flink.apache.org), [Apache Spark](http://spark.apache.org), and [Google Cloud Dataflow](https://cloud.google.com/dataflow).
```
Comment (davorbonaci) [inline src/index.md]: Can you do the same for the new [`Quickstart`](http://beam.incubator.apache.org/get-started/quickstart/) page too?

**Comment by tweise** [comment]
@davorbonaci done. Build error unrelated:
```
- ./content/subdir/beam/update/2016/10/11/strata-hadoop-world-and-beam.html
  *  External link https://www.oreilly.com/ideas/future-proof-and-scale-proof-your-code failed: got a time out (response code 0)
rake aborted
```

File: src/get-started/quickstart.md [hunk 3]
```diff
@@ -73,6 +73,12 @@ $ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
      -Dexec.args="--inputFile=pom.xml --output=counts"
 ```
 
+{:.runner-apex}
+``` 
+$ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
```
Comment (davorbonaci) [inline src/get-started/quickstart.md]: Could we teach the users here how to run on Apex locally and on a remote cluster?

File: src/get-started/quickstart.md [hunk 4]
```diff
@@ -100,6 +106,11 @@ Once the pipeline has completed, you can view the output. You'll notice that the
 $ ls counts*
 ```
 
+{:.runner-apex}
```
Comment (davorbonaci) [inline src/get-started/quickstart.md]: Is there some UI that users can see if running locally/remotely?

File: src/get-started/quickstart.md [hunk 5]
```diff
@@ -56,7 +56,7 @@ For a detailed introduction to the Beam concepts used in these examples, see the
 
 ## Run WordCount
 
-A single Beam pipeline can run on multiple Beam [runners]({{ site.baseurl }}/documentation#runners), including the [SparkRunner]({{ site.baseurl }}/documentation/runners/spark), [FlinkRunner]({{ site.baseurl }}/documentation/runners/flink), or [DataflowRunner]({{ site.baseurl }}/documentation/runners/dataflow). The [DirectRunner]({{ site.baseurl }}/documentation/runners/direct) is a common runner for getting started, as it runs locally on your machine and requires no specific setup.
+A single Beam pipeline can run on multiple Beam [runners]({{ site.baseurl }}/documentation#runners), including the [SparkRunner]({{ site.baseurl }}/documentation/runners/spark), [FlinkRunner]({{ site.baseurl }}/documentation/runners/flink), [ApexRunner]({{ site.baseurl }}/documentation/runners/apex) or [DataflowRunner]({{ site.baseurl }}/documentation/runners/dataflow). The [DirectRunner]({{ site.baseurl }}/documentation/runners/direct) is a common runner for getting started, as it runs locally on your machine and requires no specific setup.
```
Comment (davorbonaci) [inline src/get-started/quickstart.md]: same order as elsewhere: Apex, Flink, Spark, Dataflow?

File: src/get-started/quickstart.md (hunk 3 above)
Comment (tweise) [inline src/get-started/quickstart.md]: Will add it later when the runner supports launch on YARN cluster - BEAM-830

File: src/get-started/quickstart.md (hunk 4 above)
Comment (tweise) [inline src/get-started/quickstart.md]: Not for embedded mode, we will add the instructions for YARN later.

File: src/get-started/quickstart.md (hunk 5 above)
Comment (tweise) [inline src/get-started/quickstart.md]: done

**Comment by davorbonaci** [comment]
Perfect. Merging.

**Pull request merged**